tests/test_servefile.py

```diff
@@ -20,6 +20,23 @@
     connrefused_exc = socket.error
 
 
+def _get_port_from_env(var_name, default):
+    if var_name not in os.environ:
+        return default
+    port = int(os.environ[var_name])
```

Comment:
Not handling `ValueError` is probably okay in the context of testing, but the three lines can probably simplified to `port = int(os.environ.get(var_name, default))` but I might be missing something.
`

Reply:
Not handling any broken ports handed in by the user was intentionally left blank. I don't consider the test user interface to be something that needs to be polished. If it breaks the dev running the tests will see a stacktrace which tells them what's wrong.

The code looks the way it looks because I originally wanted to quit early and not run through the block below. Nothing that makes sense or warrants additional locs, so, good feedback.